reference/enterprise-api-ref/file-changes.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: File Changes API
+published: true
+tags: [reference, enterprise, API, reporting, file changes]
+---
+
+
+## File changes statistics
+
+**URI:** https://hub.cfengine.com/api/file-changes/statistics?fromTime=:fromTime&toTime=:toTime
+
+**Method:** GET
+
+Get file changes statistics by period.
+
+* **fromTime** *(timestamp)*
+    Include changes performed within interval.
```

Comment:
I see from example below what the format of (timestamp) is but I think it should be explained here. Something like YYYY-mm-dd%HHMM:SS maybe? (using the letters from `man date`).

Reply:
added